images/recycler/Dockerfile

```diff
@@ -0,0 +1,20 @@
+#
+# This is the default deployment strategy image for OpenShift Origin. It expects a set of
+# environment variables to parameterize the deploy:
+#
+#   KUBERNETES_MASTER - the address of the OpenShift master
+#   KUBERNETES_DEPLOYMENT_ID - the deployment identifier that is running this build
+#
+# The standard name for this image is openshift/origin-deployer
+#
+FROM openshift/origin
+
+RUN yum install -y nfs-utils sudo
+
+# Edit sudoers file
+# To avoid error: sudo: sorry, you must have a tty to run sudo
+RUN sed -i -e "s/Defaults    requiretty.*/ #Defaults    requiretty/g" /etc/sudoers
```

Comment:
is there a better way to do this?
how will we know in the future if this sed command fails to replace anything and leaves our recycler in a broken state?
Should we do something like this to avoid requiring an exact match of four spaces between `Defaults` and `requiretty`?
`"s/Defaults\s+requiretty.*/#Defaults requiretty/g"`